student.rb

```diff
@@ -0,0 +1,18 @@
+require_relative 'person'
+
+# Class Student inherits from Person
+class Student < Person
+  attr_reader :classroom
```

Comment:
- [ ] Kindly ensure classroom is not a publicly readable method. Always ensure that none of your attributes are readable or writable by default unless you specifically know they're supposed to be get or set by another class or it has been specified in the ERD diagram. Kindly remove this attr_reader.